how-it-works/00_General/01_overview-of-the-internet-computer.card.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Architecture of the Internet Computer
+---
+
+![](/img/how-it-works/subnet_architecture.png)
+
+# Architecture of the Internet Computer
+
+The Internet Computer implements the vision of a World Computer –  an open and secure blockchain-based network that can host all of humanity's computations in a secure and trustworthy way, witout needing to trust any single party to behave properly.
+To achieve this, the technology behind the Internet Computer (IC) had to be reconceived from ground up, learning many lessons from earlier projects that failed to create a World Computer.
+
+At a high level, the IC's architecture builds on *subnets*, or *subnetworks*: The IC as a whole consists of many subnets, where each subnet is its own blockchain that operates concurrently with and independently of the other subnets.
+A subnet is a blockchain-based *replicated state machine*, that is, a virtual machine that holds state in a secure and non-tamperable manner.
+Each subnet hosts smart contracts, called *canister smart contracts*, or just *canisters*.
+New subnets can be created from nodes added to the IC to scale out the system, analogous to how traditional architectures such as public clouds scale out by adding machines.
+Such a scale-out architecture is unique in the blockchain space and allows for limitless scaling, i.e., combining the security and resiliency properties of blockchains with the scalability properties known from the public cloud.
+
+A canister smart contract can send messages to canisters on the same or different subnets, thereby realizing interactions between canister smart contracts.
+The secure *asynchronous cross-subnet (xnet) communication* between canisters and the resulting loose coupling of subnets is one of the key principles that unlock the scalability of the IC by means of adding new subnets.
```

Comment:
The fact that the inter-canister communication may be "cross-subnet" does not seem to be too important here. I like how you link the asynchronous interaction with scaling and contrast it to other blockchains, but I'm not sure the term "loose coupling" here is well-understood.

Reply:
I'm also not sure that the term *asynchrony* is well understood here either...target audience, please!!! At least, could we add a sentence here that states what we mean and why it is important